src/wmtk/TetMeshOperationExecutor.cpp

```diff
@@ -681,7 +633,20 @@ Tuple TetMesh::TetMeshOperationExecutor::split_edge()
     // return tuple
     // which one to return?
     // TODO
-    Tuple ret = m_mesh.edge_tuple_from_id(new_eids[0]);
+    // Tuple ret = m_mesh.edge_tuple_from_id(new_eids[0]);
+    // std::cout << "return_tid: " << return_tid << std::endl;
+    // std::cout << "return_local_vid: " << return_local_vid << std::endl;
+    // std::cout << "return_local_eid: " << return_local_eid << std::endl;
+    // std::cout << "return_local_fid: " << return_local_fid << std::endl;
+
+
+    assert(return_tid > 0);
+    assert(return_local_vid > 0);
+    assert(return_local_eid > 0);
+    assert(return_local_fid > 0);
+    long return_tet_hash = hash_accessor.index_access().scalar_attribute(return_tid);
```

Comment:
Please make every variable that will never change `const`. In general, use `const` as much as possible.